src/main/resources/templates/repo/kinesis-log-streams.json

```diff
@@ -113,5 +113,13 @@
 				"day": "string"
 			}
 		}
+	},
+	{
+		"name": "aclSnapshots",
+		"partitionScheme": "!{timestamp:yyyy-MM-dd}",
+		"bucket": "${stack}.analytics.sagebase.org",

```

Comment:
Maybe a dedicated bucket for snapshots instead might be better, e.g. ${stack}.snapshots.sagebase.org. We need to be careful not to use the same "snapshot" bucket used previously.